dev-packages/application-manager/src/generator/frontend-generator.ts

```diff
@@ -188,17 +188,22 @@ app.on('ready', () => {
             width, height, x, y
         });
 
-        let windowOptions = {
-            show: false,
-            title: applicationName,
-            width: windowState.width,
-            height: windowState.height,
-            minWidth: 200,
-            minHeight: 120,
-            x: windowState.x,
-            y: windowState.y,
-            isMaximized: windowState.isMaximized
-        };
+        let windowOptions = electronStore.get('windowoptions');
```

Comment:
[minor] should it be renamed to `windowOptions` instead of `windowoptions`?

Reply:
https://github.com/eclipse-theia/theia/blob/346282bd61839ff1f9848229c354786e434b0269/dev-packages/application-manager/src/generator/frontend-generator.ts#L186

lowercase was also used here so I thought that I'll also stay with lowercase-only. But using camelCase would also be fine with me, if you prefer camelCase. I'll change it asap if you think we should use camelCase here 👍

Reply:
We generally use camelCase (as per the coding guidelines), I think `windowstate` might be a mistake.
If you don't mind, can you make the change to `windowOptions`?

Reply:
Done 👍